File: seongkyu-lim/Programmers/step1/Python/두개뽑아서더하기.py
def solution(numbers):
    answer = []
    for i in range(len(numbers)):
        for j in range(i+1, len(numbers)):
            new_num = numbers[i]+numbers[j]
            answer.append(new_num)
    answer = set(answer)
    answer = list(answer)
    answer.sort()
    return answer

File: mmkser10/javascript/programmers/두_개_뽑아서_더하기.js
function solution(numbers) {
    let set = new Set();
    for (let i = 0; i < numbers.length - 1; i++) {
        for (let j = 1 + i; j < numbers.length; j++) {
            set.add(numbers[i] + numbers[j]);
        }
    }
    let answer = Array.from(set);
    answer.sort( (x, y) => x-y);
    return answer;
}